content/en/docs/tasks/administer-cluster/namespaces-walkthrough.md

```diff
@@ -268,7 +268,8 @@ kubectl get pods
 Production likes to run cattle, so let's create some cattle pods.
 
 ```shell
-kubectl run cattle --image=k8s.gcr.io/serve_hostname --replicas=5
+kubectl create deploy cattle --image=k8s.gcr.io/serve_hostname
+kubectl scale deploy cattle --replicas=5
```

Comment:
Because this is documentation and is aimed at people new to Kubernetes, I'd prefer to avoid the shorthand:
```suggestion
kubectl create deployment cattle --image=k8s.gcr.io/serve_hostname
kubectl scale deployment cattle --replicas=5
```